content/en-us/reference/engine/classes/Explosion.yaml

```diff
@@ -8,8 +8,7 @@ summary: |
   `Class.WeldConstraint|WeldConstraints` between parts and kills
   `Class.Humanoid` characters not protected by a `Class.ForceField`.
 description: |
-  An Explosion applies force to `BaseParts|BasePart` within the explosion's
-  `Class.Explosion.BlastRadius`. This force breaks
+  An Explosion applies force to `Class.BasePart|BaseParts` within the explosion's `Class.Explosion.BlastRadius`. This force breaks
```

Comment:
```suggestion
  An Explosion applies force to `Class.BasePart|BaseParts` within the explosion's `Class.Explosion.BlastRadius|BlastRadius`. This force breaks
```

Reply:
Can use the shorthand "name only" since it's already on the Explosion class page.